### PR TITLE
Display times at /times and /times/:time

### PR DESCRIPTION
**Pull request by iCHAIT**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -27,7 +27,15 @@
 
 # get_times
 get '/times' do
-  erb :times
+  erb :times, locals: { times: ts.get_times }
+end
+
+get '/times/:time' do
+  data = []
+  ts.get_times.each do |t|
```
Comment (Kennric) [inline app.rb]: You can fetch an individual time by uuid, just include the uuid: paramter, assuming rimesync behave like pymesync

**Comment by iCHAIT** [comment]
Ah, yes, forgot about that, fixed.

**Comment by iCHAIT** [comment]
Ready for review

**Comment by Kennric** [comment]
It's a small thing, but in template tags, please put a space between the value and final %>

```
<h1 class="page-header"><%= header %></h1>
                                 ^^
```

**Comment by Kennric** [comment]
+1

**Pull request merged**